lib/std/os/test.zig

```diff
@@ -627,3 +627,19 @@ test "getrlimit and setrlimit" {
         try os.setrlimit(resource, limit);
     }
 }
+
+test "shutdown socket" {
+    if (builtin.os.tag != .linux and builtin.os.tag != .windows)
```

Comment:
NO.

Please fill the entries for [openbsd](https://github.com/openbsd/src/blob/master/sys/sys/socket.h), [netbsd](http://fxr.watson.org/fxr/source/sys/socket.h?v=NETBSD) and [freebsd](http://fxr.watson.org/fxr/source/sys/socket.h).

Reply:
ok I've filled out the entries for those platforms as well